pkg/webhooks/pytorch/pytorchjob_webhook_test.go

```diff
@@ -284,11 +290,56 @@ func TestValidateV1PyTorchJob(t *testing.T) {
 					specPath.Child("elasticPolicy").Child("nProcPerNode"), specPath.Child("nprocPerNode")),
 			},
 		},
+		"unsupported managedBy controller name": {
+			pytorchJob: &trainingoperator.PyTorchJob{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test",
+				},
+				Spec: trainingoperator.PyTorchJobSpec{
+					RunPolicy: trainingoperator.RunPolicy{
+						ManagedBy: ptr.To("other-job-controller"),
+					},
+					PyTorchReplicaSpecs: validPyTorchReplicaSpecs,
+				},
+			},
+			wantErr: field.ErrorList{
+				field.NotSupported(field.NewPath("spec", "runPolicy", "managedBy"), "", sets.List(sets.New(
+					trainingoperator.MultiKueueController,
+					trainingoperator.KubeflowJobsController))),
+			},
+		},
+		"managedBy field becomes mutable": {
```

Comment:
```suggestion
		"attempt to update the managedBy field gets rejected": {
```
same for other tests